mmdet/evaluation/metrics/coco_metric.py

```diff
@@ -96,6 +105,10 @@ def __init__(self,
         self.classwise = classwise
         # whether to use multi processing evaluation, default False
         self.use_mp_eval = use_mp_eval
+        # whether to use Faster Coco Eval, default False
+        self.use_faster_coco_eval = use_faster_coco_eval
+        if FasterCOCO is None:
```

Comment:
Seems this may get error when the user not installed FasterCOCO

```suggestion
        if self.use_faster_coco_eva:
            assert FasterCOCO is not None, 'faster-coco-eval is not installed'
```